dawn/src/dawn/SIR/proto/SIR/statements.proto

```diff
@@ -545,7 +563,12 @@ message FieldAccessExpr {
   // if we have directional or offset arguments, we have to perform a lazy evaluation to compute
   // the real offset once we know the mapping of the directions (and offsets) to the actual
   // arguments of the stencil function.
-  repeated int32 offset = 2;
+  int32 vertical_offset = 2;
```

Comment:
For symmetry with `Extents`, would it make sense to introduce `Offsets`. No strong opinion.

Reply:
My feeling is no. But I am not a protobuf expert.  I think accessing this would look less nice:
```
myExpr.vertical_offset().value()
```
vs
```
myExpr.vertical_offset()
```
and we would also need to introduce it at other places (like for the argument argument offsts).